packages/autorest.python/autorest/preprocess/__init__.py

```diff
@@ -30,10 +30,13 @@ def add_body_param_type(code_model: Dict[str, Any], body_parameter: Dict[str, An
         and not body_parameter["type"].get("xmlMetadata")
         and not any(t for t in ["flattened", "groupedBy"] if body_parameter.get(t))
     ):
+        origin_type = body_parameter["type"]["type"]
         body_parameter["type"] = {
             "type": "combined",
             "types": [body_parameter["type"], KNOWN_TYPES["binary"]],
         }
+        if origin_type == "model":
```

Comment:
why do we need any type for models?

Reply:
The 'any-type' is generated as JSON in our codegen, here it's for generating an overload with body type of "JSON"